app/controllers/submissions_controller.rb

```diff
@@ -11,9 +11,11 @@ def create
     if @submission.valid?
       if @submission.passed_screening?
         deposit_agreement_path = Submissions::DocumentDepositAgreement.call(@submission.submitter)
+        deposit_instructions_path = Submissions::CreateDepositInstructions.call(@submission)
```

Comment:
Looks like there's a mismatch between the arguments passed to this `call` method and the parameters expected by the service class -- `@submission` vs. `user`.  I'm actually not sure you need any arguments/parameters at all for this service?